index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Porfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;500;600;700;800&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./assets/style/style.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="nav-container">
+            <div class="nav-text">Welcome</div>
+            <div class="hamburgur-menu">
+                <img class="hamburger" src="./assets/icons/hamburger.png" alt="menu-hamburger">
+            </div>
+        </div>
+    </nav>
+
+    <div class="headline-container">
+
+        <div class="headline">
+            <div class="primary-headline">
+                <p class="primary-headline-text">Hey there. I’m Ansderly. <br>
+                    <span class="primary-headline-text-second-color"> I’m a software developer</span>
+                </p>
+            </div>
+            <div class="supporting-text-container">
+                <p class="supporting-text"> I can help you build a product , feature or website Look through some of my
+                    work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+            </div>
+            <div class="social-media-container">
+                <img class="social-media" src="./assets/images/social media icons.svg" alt="social-media-icon">
+            </div>
```

Comment:
- [ ] Kindly ensure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`.